src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -804,6 +806,56 @@ public Response deleteProject(
         }
     }
 
+    @POST
+    @Path("/batchDelete")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Deletes a list of projects specified by their UUIDs",
+            description = "<p>Requires permission <strong>PORTFOLIO_MANAGEMENT</strong></p>"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "204", description = "Projects removed successfully"),
+            @ApiResponse(responseCode = "207", description = "Access is forbidden to the projects listed in the response"),
+            @ApiResponse(responseCode = "401", description = "Unauthorized")
+    })
+    @PermissionRequired(Permissions.Constants.PORTFOLIO_MANAGEMENT)
+    public Response deleteProjects(List<UUID> uuids) {
```

Comment:
Consider using `@Size(min = 1, max = ...)` here. The limits should then also appear in the OpenAPI spec automatically.

I think there should be *some* upper limit to the number of UUIDs being submitted in one go. We can always extend it later if people need it, but we can't reduce it later if larger batches end up causing problems.

Reply:
That's a good idea. I have tested it with up to 1000 UUIDs, so maybe that could be a reasonable max size to start with?

Reply:
Yeah that sounds reasonable.